clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/safety/ClusterMatchRule.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.netflix.spinnaker.clouddriver.safety;
+
+public class ClusterMatchRule {
```

Comment:
I would've Lombok'd this class, but given the odd handling of null values, I didn't feel comfortable making those changes.

Reply:
Yeah, looks like it could use some `null -> empty` translation in the setters, but agree that's a separate issue from the changes here.